src/main/kotlin/com/pubnub/api/models/consumer/access_manager/v3/Grants.kt

```diff
@@ -37,7 +40,19 @@ internal data class PNChannelResourceGrant(
     override val get: Boolean = false,
     override val join: Boolean = false,
     override val update: Boolean = false
-) : PNResourceGrant(), ChannelGrant
+) : PNResourceGrant(), ChannelGrant, SpaceIdGrant {
+    constructor(spaceIdGrant: SpaceIdGrant) : this(
```

Comment:
I don't think I like that old classes knows anything about new classes

Reply:
ok, I will create set of new classes